.github/workflows/lima.yaml

```diff
@@ -13,17 +13,7 @@ permissions:
 jobs:
   integration:
     name: Integration tests
-    strategy:
-      fail-fast: false
-      matrix:
-        # macos-13-large is used as macos-13 seems too flaky.
-        # macos-14 (ARM) and later cannot be used for the most part of the job
-        # due to the lack of the support for nested virt.
-        platform:
-          - macos-13-large
-          - macos-14-large
-          - macos-15-large
-    runs-on: ${{ matrix.platform }}
+    runs-on: macos-15-large
```

Comment:
I would keep `{{ matrix.platform }}` and a list with one item so adding more items is easier later. This also generates more useful name for the job (integration (macos-15)).

Reply:
fixed